src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -206,4 +211,28 @@ const PathwayVisualizer: FC = () => {
   );
 };
 
+const McodeVisualizer: FC = () => {
+  const mcode = usePatientRecords().mcodeRecords;
+  console.log(mcode);
+  type mcodeKey = keyof McodeElements & string;
+  const keysArray: mcodeKey[] = [];
+  for (const key in mcode) {
+    keysArray.push(key as mcodeKey);
+  }
+  return (
+    <table>
+      <tbody>
+        {keysArray.map(key => {
+          return (
+            <tr key={key}>
+              <td>{key}</td>
```

Comment:
We should consider formatting the key here. Without introducing any other data a simple camel case convertor could work - capitalize the first character and then add a space before each capital letter